Firmware/power_panic.cpp

```diff
@@ -163,10 +163,8 @@ void uvlo_() {
         eeprom_update_word((uint16_t*)(EEPROM_UVLO_MESH_BED_LEVELING_FULL +2*mesh_point), *reinterpret_cast<uint16_t*>(&v));
     }
 
-    // Write the _final_ Z position and motor microstep counter (unused).
+    // Write the _final_ Z position
     eeprom_update_float((float*)EEPROM_UVLO_TINY_CURRENT_POSITION_Z, current_position[Z_AXIS]);
-    z_microsteps = tmc2130_rd_MSCNT(Z_AXIS);
-    eeprom_update_word((uint16_t*)(EEPROM_UVLO_Z_MICROSTEPS), z_microsteps);
```

Comment:
Are you sure this part was never used? In the past, there were issues with PP not recovering the Z-height correctly in some cases. @wavexx spent a lot of time debugging that.

Reply:
Yes at least not by the firmware.  We are not reading from this EEPROM address anywhere. It was only being written to.